@kiva/kv-components/vue/KvLoanTag.vue

```diff
@@ -55,6 +57,10 @@ export default {
 			return null;
 		},
 		tagText() {
+			const partnerName = this.loan?.partnerName ?? '';
+			if (partnerName.includes(LSE_LOAN_KEY)) {
+				return 'High community impact';
```

Comment:
So we're showing both a custom tag and callout for the LSE loans? Do you have a screenshot example from our dev data?

Reply:
We are just showing a tag for LSE loans.
For callouts we are just changing the priority order to have the tag first if exists and just showing one callout.
This is the priority order for LSE loans 1.Tag, 2. Category, 3. Activity, 4. Sector, 5. Theme instead of 1. Category, 2. Activity, 3. Sector, 4. Tag, 5. Theme. for others